modules/azure/cosmosdb.go

```diff
@@ -0,0 +1,185 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/profiles/preview/cosmos-db/mgmt/documentdb"
+	"github.com/gruntwork-io/terratest/modules/testing"
+	"github.com/stretchr/testify/require"
+)
+
+// GetCosmosDBAccountClient is a helper function that will setup a CosmosDB account client.
+func GetCosmosDBAccountClient(subscriptionID string) (*documentdb.DatabaseAccountsClient, error) {
```

Comment:
In Terratest almost all methods support foo and fooE methods to allow for better error handling.  Pattern is documented [here](https://terratest.gruntwork.io/docs/testing-best-practices/error-handling/), please follow this pattern also.

Reply:
Done.